javascript/packages/orchestrator/src/constants.ts

```diff
@@ -44,6 +44,11 @@ const FINISH_MAGIC_FILE = "/tmp/finished.txt";
 const GENESIS_STATE_FILENAME = "genesis-state";
 const GENESIS_WASM_FILENAME = "genesis-wasm";
 
+// TODO: replace ${} with {{ZOMBIE:RAWCHAINSPEC}} etc. for consistent style
```

Comment:
I think its good to add this TODO in the same PR

Reply:
Absolutely, will do, left it there for any possible comments before changing to this format